### DISCUSSION TITLE
Bizarre error with VS Code when refactoring with F2 and SvelteKit running within a Docker container (with strange proxy+pager.server.ts files)

### DISCUSSION THREAD

**Discussion by 205g0** [sveltejs/kit#8906] [Q&A]
Then, VS Code tries also to change and save files in `.svelte-kit` (i.e. `proxy+page.server.ts`) which seem to be affected by the refactoring.

This, of course, doesn't work because `.svelte-kit` is a folder created by Docker and `root` which VS Code can't and shouldn't write to. Now, I've tried everything to stop VS Code from doing this (tested all possible `exclude` settings) but nothing worked.

Anyone knows more about these `proxy...` files?

**Comment by david-plugge** [comment]
The .svelte-kit folder is created by sveltekit. The contents are auto generated stuff for types (so you can use `./$types` imports) and other configuration. It's expected behavior.

**Reply by 205g0** [reply]
Thanks, I know but why does VS Code try to refactor in this folder as well and how can this be disabled?
